lsp-test/test/DummyServer.hs

```diff
@@ -185,4 +185,21 @@ handlers =
                 Nothing
                 Nothing
         resp $ Right $ InR res
+     , requestHandler STextDocumentPrepareCallHierarchy $ \req resp -> do
```

Comment:
Should we test the other methods too?

Reply:
We should, I jumped to hls development and missed the other two, although it works well.